graders/r/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Borrow the leg work done by the rocker project
+FROM rocker/tidyverse:3.6.1
+
+# Set a new label for the image
+LABEL org.label-schema.license="GPL-2.0" \
+      org.label-schema.vcs-url="https://github.com/PrairieLearn/PrairieLearn" \
+      org.label-schema.vendor="PrairieLearn" \
+      maintainer="James Balamuta <balamut2@illinois.edu>"
+
+# Add a script to install any additional R packages
+COPY r-requirements.R /
+
+# Modify the image to meet with PrairieLearn specifications
+RUN useradd ag && \                        # Enable autograde user for PrairieLearn
+    su root -c "Rscript /r-requirements.R" # Install the additional packages
```

Comment:
Have you tested this? I think the comment on line 14 will cause line 15 to not execute.

Reply:
Mmm, I did test the image creation. You can see that the `Dockerfile` successfully built: 

https://cloud.docker.com/repository/registry-1.docker.io/jamesbalamuta/ubuntu-pl-r/builds/f6ade2cb-e3a5-4930-a73a-b9ac50bbd4dd

However, I didn't test whether `RcppArmadillo` was enabled. I'll setup something that uses commenting from the `centos-plbase` image: 

https://github.com/PrairieLearn/PrairieLearn/blob/master/environments/centos7-plbase/Dockerfile#L53

Reply:
The change to comment format fixes this, thanks!